Getting-Started/Setup/Server-Setup/Load-Balancing/index-v7.md

```diff
@@ -0,0 +1,103 @@
+* .NET Framework
+
+:::note
+It is highly recommended that you setup your staging environment to also be load balanced so that you can run all of your testing on a similar environment to your live environment.
+:::
+
+## Flexible load balancing
+
+With Umbraco version 7.3.0, load balancing is easier than ever before.
+Setting up a load balanced environment is achievable with a few configuration file changes.
+
+[Full documentation is available here](flexible.md)  
+
+## Legacy load balancing
+
+Umbraco versions before 7.3.0 must use a legacy/deprecated load balancing technique.
+
+[Full documentation is available here](traditional.md)  
+
+## Common load balancing setup information
+
+_The below section is common for all load balancing configurations, ensure that the following is instructions are followed regardless of what load balancing setup you choose._
+
+### ASP.NET Configuration
+
+* You will need to use a custom machine key so that all your machine key level encryption values are the same on all servers, without this you will end up with view state errors, validation errors and encryption/decryption errors since each server will have its own generated key.
+	* Here are a couple of tools that can be used to generate machine keys:
+		* 	[Tool 1: http://www.betterbuilt.com/machinekey/](http://www.betterbuilt.com/machinekey/)
+		* 	[Tool 2: http://www.developerfusion.com/tools/generatemachinekey/](https://www.developerfusion.com/tools/generatemachinekey/)
+	* 	Then you need to update your web.config accordingly, note that the validation/decryption types may be different for your environment depending on how you've generated your keys.
+
+```xml
+<configuration>
+  <system.web>
+    <machineKey decryptionKey="Your decryption key here"
+                validationKey="Your Validation key here"
+                validation="SHA1"
+                decryption="AES" />
+  </system.web>
+</configuration>
+```
+            
+* If you use SessionState in your application, or are using the default TempDataProvider in MVC (which uses SessionState) then you will need to configure your application to use the SqlSessionStateStore provider (see [https://msdn.microsoft.com/en-us/library/aa478952.aspx](https://msdn.microsoft.com/en-us/library/aa478952.aspx) for more details).
+
+### Logging
+
+There are some logging configurations to take into account no matter what type of load balancing environment you are using.
+
+[Full documentation is available here](logging.md)
+
+### Testing
+
+Your staging environment should also be load balanced so that you can see any issues relating to load balancing in that environment before going to production.
+
+You'll need to test this solution **a lot** before going to production. You need to ensure there are no windows security issues, etc... The best way to determine issues is have a lot of people testing this setup and ensuring all errors and warnings in your application/system logs in Windows are fixed.
```

Comment:
This doesn't seem to come across as good advice, although the sentiment is correct... it just says 'this is really flakey and hard to set up' but that shouldn't be the message if you follow the instructions.

Could be replaced with a link to the community Load Balancing healthcheck for V7, and V8 version (umm when that's released)... but also perhaps suggest a way of testing...

1) Update content on Master
2) Check update appears on Master Server
3) check update appears on Replica servers
If using Cloud scale up Replica environments and try again
Try Scheduled publishing of content
Check and review application/system logs for errors